uint32[2] node_indices
# index into materials- the materials need to already
# exist for this to be useful, though 0 is a safe default
uint8 material_ind
